chpl/chpl-resources/src/main/resources/errors.properties

```diff
@@ -345,6 +345,7 @@ listing.duplicateTestingLab=Listing contains duplicate Testing Lab: '%s'. The du
 listing.criteria.duplicateTestTool=Certification %s contains duplicate Test Tool: Name '%s', Version '%s'. The duplicates have been removed.
 listing.criteria.duplicateTestFunctionality=Certification %s contains duplicate Test Functionality: Number '%s'. The duplicates have been removed.
 listing.criteria.duplicateTestStandard=Certification %s contains duplicate Test Standard: Number '%s'. The duplicates have been removed.
+listing.criteria.required=Criteria %s is required.
```

Comment:
The previous error messages related to criteria called them "Certification %s". I'm not against changing the pattern, but if we do we probably should for all of them